src/i18n/en.json

```diff
@@ -1,4 +1,20 @@
 {
+  "header_nav": {
```

Comment:
I think we don't need to have the menu translation here, right? Since we have it in the json model

Reply:
yes good catch! thanks!